back/engines/commercial/bulk_import_ideas/app/views/bulk_import_ideas/web_api/v1/export_form.html.erb

```diff
@@ -130,6 +142,32 @@
               <% end %>
             </div>
 
+          <% elsif field[:format] == :multi_select_image %>
+            <div class="image-select">
+              <% field[:options].each do |option| %>
+                <div>
+                  <div class="image">
+                    <% if option[:image_url] %>
+                      <img alt="<%= option[:title] %>" src="<%= option[:image_url] %>" />
+                    <% else %>
+                      <!-- No image placeholder -->
```

Comment:
Used SVG partly as getting to the assets that we use in FE is a little difficult, but also it's tiny and will look crisper in print